Level1.java

```diff
@@ -1,22 +1,51 @@
 import greenfoot.*;
 
-public class Level1 extends World
+public class Level1 extends General
 {
     public Level1()
     {
-        super(900, 500, 1); 
         addObject (new CellCharacter(),450,150);
-        createVirus(20);
+        createVirus(15);        
+        prepare();
     }
-    
+
     public void createVirus(int quantity) {
         int x, y;
-        
+
         for(int i = 0; i < quantity; i++) {
             VirusCharacter virus = new VirusCharacter();
             x = Greenfoot.getRandomNumber(getWidth());
             y = Greenfoot.getRandomNumber(getHeight());
             addObject(virus, x, y);
         }
     }
+    
+    /**
+     * Prepare the world for the start of the program.
+     * That is: create the initial objects and add them to the world.
+     */
+    private void prepare()
+    {
+        Donut donut = new Donut();
```

Comment:
pudieron ahorrarse todas las líneas donde declaran donuts y solo usar  
```
addObject(new Donut(),109,64
```

El objetivo no es tener código pequeño, pero también debemos evitar crear código inútil, nos obliga a leer más..